internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -461,7 +462,7 @@ public RssSendShuffleDataResponse sendShuffleData(RssSendShuffleDataRequest requ
             maxRetryAttempts,
             t -> !(t instanceof OutOfMemoryError));
       } catch (Throwable throwable) {
-        LOG.warn(throwable.getMessage());
+        LOG.warn(ExceptionUtils.getStackTrace(throwable));
```

Comment:
ditto

Reply:
ditto

`LOG.warn("Failed to require pre-allocated memory or send shuffle data due to ", throwable);`
or
`LOG.warn(ExceptionUtils.getStackTrace(throwable));`

Reply:
`LOG.warn("Failed to require pre-allocated memory or send shuffle data due to ", throwable);` +1